### PR TITLE
[Release][Tensorflow]GPU py27 and py37 Images

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*
Releasing GPU py27 and py37 Images for TF1.15

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release_images.yml [hunk 1]
```diff
@@ -4,31 +4,41 @@ release_images:
...
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
+      version: "2.2.0"
+      training:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu101"
+        example: False        # [Default: False] Set to True to denote that this image is an Example image
+          disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```
Comment (YYStreet) [inline release_images.yml:25]: The indentation is incorrect here

Comment (TusharKanekiDey) [reply release_images.yml]: Thanks, made the change.

**Comment by TusharKanekiDey** [comment]
> Should we also include the part of tf2.2 for tomorrow morning's release in this PR?

Yes, We need to include tf2.2 release logic as well.

File: release_images.yml [hunk 2]
```diff
@@ -4,31 +4,41 @@ release_images:
...
+        disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+        # to images being published.
+        force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+        # has already been published. Re-released image will have minor version incremented by 1.
+      inference:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu102"
+  3:
```
Comment (saimidu) [inline release_images.yml:34]: Add the `disable_sm_tags: False` and `force_release: False` options here for the TF 2.2 inference image.

**Comment by saimidu** [comment]
Thanks for the PR.

**Comment by TusharKanekiDey** [comment]
> Thanks for the PR.

Thanks for the approval.

File: release_images.yml [hunk 3]
```diff
@@ -23,12 +23,34 @@ release_images:
...
+        example: False        # [Default: False] Set to True to denote that this image is an Example image
+        disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+        # to images being published.
+        force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+        # has already been published. Re-released image will have minor version incremented by 1.
+      inference:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu102"
```
Comment (arjkesh) [inline release_images.yml:41]: why are mismatched cuda versions being released?

File: release_images.yml [hunk 4]
```diff
@@ -4,14 +4,14 @@ release_images:
...
-      python_versions: ["py37", "py36", "py27"]
+      device_types: ["gpu"]
+      python_versions: ["py37", "py27"]
       os_version: "ubuntu18.04"
       cuda_version: "cu100"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
                             # to images being published.
-      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+      force_release: True  # [Default: False] Set to True to force images to be published even if the same image
```
Comment (arjkesh) [inline release_images.yml:14]: do we not need to re-release example images?

File: release_images.yml [hunk 5]
```diff
@@ -23,12 +23,34 @@ release_images:
...
-      cuda_version: "cu100"
-      example: True
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
+      version: "2.2.0"
+      training:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu101"
```
Comment (arjkesh) [inline release_images.yml:31]: why is cu101 being released for training but cu102 for inference?

File: release_images.yml (hunk 3 above)
Comment (arjkesh) [reply release_images.yml:41]: ans: cu102 is the only option for tf inference, and cu101 is the default for training

**Pull request merged**